keyboards/converter/hp_46010a/keymaps/default/keymap.c

```diff
@@ -34,7 +34,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
      * `-----------------------------------------------------------------------------------'    `-------------------'
      */
      
-    [0] = LAYOUT(
+    [0] = LAYOUT (
```

Comment:
This file should be reset.

`git checkout master -- keyboards/converter/hp_46010a/keymaps/default/keymap.c`